docker-compose.yml

```diff
@@ -100,6 +100,8 @@ services:
       - ./submodules/datapusher:/var/www/datapusher
     ports:
       - "8800:8800"
+    environment:
+      DATAPUSHER_MAX_CONTENT_LENGTH: ${DATAPUSHER_MAX_CONTENT_LENGTH}
```

Comment:
This doesn't look correct:
1. It's missing `-` at the start
2. It'd look better if you configure it to pass already existing env (just like with `CKAN_SITE_URL`)
```
environment:
  - DATAPUSHER_MAX_CONTENT_LENGTH
```
More info in docs: https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-environment-attribute

Reply:
Thanks